platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/widgets/MyLocationView.java

```diff
@@ -546,14 +547,19 @@ public void setContentPadding(int[] padding) {
     contentPaddingY = (padding[1] - padding[3]) / 2;
   }
 
-  private static class GpsLocationListener implements LocationListener {
+  private static class GpsLocationListener implements LocationEngineListener {
 
     private WeakReference<MyLocationView> userLocationView;
 
     GpsLocationListener(MyLocationView myLocationView) {
       userLocationView = new WeakReference<>(myLocationView);
     }
 
+    @Override
+    public void onConnected() {
+      // Nothing
```

Comment:
We should add integration here for showing the last known location. This would solve the regression in https://github.com/mapbox/mapbox-gl-native/issues/7796. This is something that broke with upgrading to LOST 2.0 (where the onConnected callback was introduced).